src/plugincatalog.html

```diff
@@ -7,7 +7,7 @@
 
                     <div class="selectContainer">
                         <select is="emby-select" id="selectSystem" label="${LabelDisplayPluginsFor}">
-                            <option value="Server">${HeaderEmbyServer}</option>
+                            <option value="Server">${HeaderJellyfinServer}</option>
```

Comment:
Did you check that this localizes properly? I don't see corresponding localization files' change.

Reply:
'Emby' was replaced with 'Jellyfin' in all strings very early on -- even keys

Reply:
Just checked, it's included in the strings.

Reply:
@JustAMan i checked it it's already there "HeaderJellyfinServer": "Jellyfin Server"